tests/integration/test_vulnerability_detector/test_feeds/test_invalid_values_redhat_feeds.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import json
+from time import sleep
+import itertools
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import truncate_file, read_file, write_file
+from wazuh_testing.vulnerability_detector import CUSTOM_NVD_VULNERABILITIES_1, CUSTOM_NVD_FEED, \
+                                                 REDHAT_KEY_FIELDS_FEEDS, set_custom_system, \
+                                                 check_vulnerability_event, check_vulnerabilities_number, \
+                                                 clean_vuln_and_sys_programs_tables
+from wazuh_testing.tools.services import control_service
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
+
+# Variables
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, 'data')
+configurations_path = os.path.join(test_data_path, 'configuration', 'wazuh_invalid_redhat_feeds.yaml')
+vulnerabilities_data_path = os.path.join(test_data_path, 'vulnerabilities', CUSTOM_NVD_VULNERABILITIES_1)
+custom_nvd_json_path = os.path.join(test_data_path, 'feeds', CUSTOM_NVD_FEED)
+custom_redhat_oval_feed_path = os.path.join(test_data_path, 'feeds', 'custom_redhat_oval_feed.json')
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# Set configuration
+parameters = [{'NVD_JSON_PATH': custom_nvd_json_path, 'REDHAT_JSON_PATH': custom_redhat_oval_feed_path}]
+metadata = [{'nvd_json_path': custom_nvd_json_path, 'redhat_json_path': custom_redhat_oval_feed_path}]
+ids = ['REDHAT_configuration']
+
+# Read JSON data template
+with open(vulnerabilities_data_path, 'r') as f:
+    nvd_vulnerabilities = json.loads(f.read())
```

Comment:
```suggestion
with open(vulnerabilities_data_path) as f:
    nvd_vulnerabilities = json.loads(f.read())
```
By default, `open` always uses _read_ mode.

Reply:
Done in ab1c229f9625689d173018ae41d5437228ded6be